builder/virtualbox/common/step_port_forwarding_test.go

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package common
+
+import (
+	"reflect"
+	"testing"
+)
+
+const validSemver = "v7.0"
+const invalidSemver = "v7.0.0abcd"
+
+func TestValidSemver(t *testing.T) {
```

Comment:
I'm not sure it's relevant to add this test since the code that does the conversion comes from another package, so as long as the dependency does the canonicalisation correctly, we shouldn't have any problem.